python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,25 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+# Load configs from servo-tidy.toml if config file exists
+if os.path.exists("./servo-tidy.toml"):
```

Comment:
``set_config`` was meant for override default configs. Top-level logic was to set constants for ignored files/dirs, But now its added to ``config`` under ``ignore``.